src/IQFeed.CSharpApiClient.Tests/Streaming/Level1/Dynamic/Messages/UpdateSummaryDynamicMessageTypesFactoryTests.cs

```diff
@@ -130,6 +133,7 @@ public void Should_Generate_Only_Specified_Properties_For_Partial_Fields()
             Assert.AreEqual(default(int), typedEmptyInstance.MostRecentTradeSize);
             Assert.AreEqual(default(DateTime), typedEmptyInstance.MostRecentTradeDate);
             Assert.AreEqual(default(TimeSpan), typedEmptyInstance.MostRecentTradeTime);
+            Assert.AreEqual(default(int), typedEmptyInstance.MostRecentTradeAggressor);
```

Comment:
shall we verify the `DynamicFieldset.MostRecentTradeDayCode` as well to ensure it doesn't throw?

Reply:
Sure. I'll put that in.